co-operative.md

```diff
@@ -0,0 +1,50 @@
+# Hypha Worker Co-operative
+
+## About Us 
+
+Hypha Worker Co-operative Inc. is a worker co-operative incorporated on August 
+1, 2019 in Ontario, Canada (Ontario Corporation No. 5019866).
+
+Hypha is incorporated as a **non-share capital** worker co-operative under the 
+[*Ontario Co-operative Corporations Act, 1990*][coop-act]. This entails specific 
+conditions on our primary object, size, and role of members, as addressed in 
```

Comment:
```suggestion
conditions on our primary objectives, size, and role of members, as addressed in 
```
Can we not use "object"? Feels like hard-to-grok jargon from the article forms.

Also, it's not immediately obvious to me what we mean by "size" :)

Reply:
"Object" is semi-official terminology from the act.

Could clarify size to "Minimum size"

Reply:
I know, but I'm not big on matching legal lingo as a general practice. Would rather favour plain language aspirations (e.g., how a normal person speaks) internally, not waiting for bureaucracy to catch up.

Ok, I understand what you mean by "size" now. Does the above revision look ok?

Reply:
I find it personally helpful to use jargon (sparingly) in moments like this as a construct to aid in recall for future tasks. 

I.e. I'm imagining the case of needing to followup later on our articles of incorporation... if we use the same critical words here as show up there (but those that are still legible to use as readers outside that context) that links/draws a connection without having to re-translate that "plain language aspirations" to the work we'll have to do.

`primary object` is so close to `primary objective` I'm not going to fight it. But I had good reasons for continuing that language while moving the rest as close to plain language as possible.

Reply:
ok! thanks! not totally aligned, but respect the consideration, and moved back to your intended language :)